src/Acuminator/Acuminator.Vsix/Coloriser/Roslyn/ParsedDocument.cs

```diff
@@ -16,9 +16,6 @@ namespace Acuminator.Vsix.Coloriser
 {
     public class ParsedDocument
     {
-        private static MetadataReference PXDataReference { get; } = 
```

Comment:
@SENya1990 Is it important reference?

Reply:
Yes it is, the coloring won't work without it.

Reply:
The code and references in VSIX project won't affect the nuget package with analyzers. We can live with the dependencies in them